QStore4S/src/main/java/edu/asu/qstore4s/repository/Element.java

```diff
@@ -0,0 +1,34 @@
+package edu.asu.qstore4s.repository;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.data.neo4j.conversion.Result;
+import org.springframework.data.neo4j.support.Neo4jTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * 
+ * @author Rachita Satyasi
+ *
+ */
+
+@Service
+public class Element {
```

Comment:
Let's call this something more descriptive (e.g. ElementNeo4jConnector or something like that).

Also, it needs an interface.